src/firmware/app/primitives/primitive.h

```diff
@@ -65,3 +65,21 @@ typedef struct
     {                                                                                    \
         free((STATE_TYPE*)state);                                                        \
     }
+
+/**
+ * Stop the robot by disabling all motors and disabling autokicking and autochipping
+ *
+ * NOTE: stop robot does not discharge capacitors
+ *
+ * @param world [in] The world containing the robot to stop
+ * @param coast Whether the robot should coast when stopping (instead of braking)
```

Comment:
``` suggestion
 * @param coast [in] Whether the robot should coast when stopping (instead of braking)
```

Reply:
this is the code style convention:
* Functions that return values via argument(s) must have all **pointer** parameters labelled as `[in]`, `[in/out]`, or `[out]` in the javadoc, **in that order**. The exceptions to this are [pseudo-class](/docs/firmware-architecture-and-design.md#pseudo-class) functions, which should take the "class" (which is an `[in/out]` as the first argument) and function pointers.
so since coast is not a pointer type, it's assumed to be [in]

Reply:
Great point, totally agree.